docs/src/add-services/influxdb.md

```diff
@@ -28,41 +28,36 @@ The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](../
 
 {{% endpoint-description type="influxdb" /%}}
 
-```php
-<?php
-// This assumes a fictional application with an array named $settings.
-if (getenv('PLATFORM_RELATIONSHIPS')) {
-	$relationships = json_decode(base64_decode($relationships), TRUE);
-
-	// For a relationship named 'influxtimedb' referring to one endpoint.
-	if (!empty($relationships['influxtimedb'])) {
-		foreach ($relationships['influxtimedb'] as $endpoint) {
-			$settings['influxdb_host'] = $endpoint['host'];
-			$settings['influxdb_port'] = $endpoint['port'];
-			break;
-		}
-	}
-}
-```
+{{< codetabs >}}
+
+---
+title=PHP
+file=static/files/fetch/examples/php/influxdb
+highlight=php
+---
+
+{{< /codetabs >}}
 
 ## Exporting data
 
-InfluxDB includes its own [export mechanism](https://docs.influxdata.com/influxdb/v1.2/tools/influx_inspect/).
-To gain access to the server from your local machine open an SSH tunnel with the Platform.sh CLI:
+To export your data from InfluxDB:
 
-```bash
-platform tunnel:open
-```
+1. Install the [influx-cli](https://docs.influxdata.com/influxdb/cloud/tools/influx-cli/).
+2. Gain access to the server from your local machine with the [Platform.sh CLI](../administration/cli/_index.md):
 
-That opens an SSH tunnel to all services on your current environment and produce output like the following:
+	```bash
+	platform tunnel:open
```

Comment:
:question: Why not use a single one? Why open all services?

Reply:
Because I thought it adds more complexity to add an extra flag when the default is okey too.